app/src/main/res/values/dimens.xml

```diff
@@ -593,4 +593,13 @@
    
   <!-- StoryFragment -->
   <dimen name="story_fragment_padding_bottom">160dp</dimen>
+
+<!--  Revision Card Fragment-->
```

Comment:
@ArchitJain1201 write name of file without spacing as StoryFragment is written

Reply:
changes was made